### ISSUE TITLE
Bill of Materials  

### ISSUE THREAD

**Issue by GaryLazerFinz** [GaryLazerFinz/ECE-411-Capstone-Practicum#11]
If we order something for the project lets put it here so we can add up the costs easily.

**Comment by bornhor2** [comment]
Should we just order 10 of everything? 10 AtTiny chips. 10 temp sensors.  We can probably get the basic stuff at school... wires and resistors.  How are we going to build the fan? Or use prebuilt fan like a computer fan?

**Comment by GaryLazerFinz** [comment]
We should definitely  order this to develop our prototype.
1528-1072-ND digikey part number
What do you all think?

**Comment by bornhor2** [comment]
When I click on these digikey links it sends me to the search page so I can't really tell what this is.
Maybe just post the part number along with it.

**Comment by GaryLazerFinz** [comment]
We will need a few transistors to power the fan.
MMBT3904LT3GOSCT-ND
 digikey patr number

**Comment by GaryLazerFinz** [comment]
I fixed it. On second thought though it would probably a good idea to order the through hole kind  of parts  to prototype  with . That would be more inexpensive .

**Comment by GaryLazerFinz** [comment]
We need to get our Bill of Materials ready.  What do we need?

**Comment by bornhor2** [comment]
We still need to get some through hole or breadboard ATTiny85s to prototype with.  If I remember correctly Ting and Xiaoqiao said that they had some?  Once we have the full circuit and software tested with through hole then we can just order the same parts with surface mount and work on our final Eagle cad design to send out.

**Comment by GaryLazerFinz** [comment]
we need to decide on a temp sensor also. i tested the ntc. it seems to
work,but doesnt have a big voltage change. I ordered some more parts:
AtTiny85 through holes, 1k resistors smd, 100k smd ntcs and 470k smd ntcs.
I bet we could get a blank copper board from the epl and cut it to test
some things.

**Comment by bornhor2** [comment]
I have one.  When I find it I'll post it on here and we can test it out.  I have the code to convert it to temp but the ATTiny programmer doesn't have a serial data stream so we'll have to use the built in LED to test when it reaches certain temperatures.

**Comment by ting6** [comment]
I have three ATtiny85. Mitch,have you ordered already?

**Comment by GaryLazerFinz** [comment]
Yeah ordered some stuff today.i kept it small to get the cheap shipping. I
think small orders under 8 oz are best. Lets make another. What do we need?

**Comment by bornhor2** [comment]
I have another midterm on Tuesday evening and a project due today but after that I will be free the rest of the week if you guys want to meet up.  I'm willing to work on this the rest of the week... we still need to put together a full eagle schematic to send off for our board and figure out what we need to turn in for the homework on Thursday.
